src/ngraph_builder.cc

```diff
@@ -131,10 +131,19 @@ static tf::Status GetInputNode(const Builder::OpMap& ng_op_map,
 
   tf::Node* tf_input;
   TF_RETURN_IF_ERROR(op->input_node(input_idx, &tf_input));
+  const std::vector<shared_ptr<ng::Node>>* ng_op = nullptr;
```

Comment:
Why does `ng_op` need to be a pointer?

Reply:
because it cannot be a reference if it gets set later.

Reply:
Oooh okay, and that assignment has to be scoped inside the `try` so we can't set it at initialization time without makin' a mess. Works for me.